python/dgl/partition.py

```diff
@@ -592,5 +592,21 @@ def map_to_global(self, idxs, part_id):
             )
         )
 
+    def generate_permutation(self, idxs):
+        """Create a mapping for the given indices to different partitions,
+        and a count of the number of indices per part.
```

Comment:
Produce a scheme that maps the given indices to separate partitions and the counts of how many indices are in each partition.

And parameter section and examples?

Reply:
Updated.